ocp_resources/job.py

```diff
@@ -63,3 +82,24 @@ def to_dict(self):
                     self.res["spec"]["template"]["spec"][
                         "restartPolicy"
                     ] = self.restart_policy
+
+    def delete(self, wait=False, timeout=TIMEOUT_4MINUTES, body=None):
+        """
+        Delete Job
```

Comment:
This is not a delete job; please change.

Reply:
Will change to "Delete Job object".